pkg/crc/services/dns/dns.go

```diff
@@ -118,10 +118,11 @@ func CheckCRCPublicDNSReachable(serviceConfig services.ServicePostStartConfig) (
 	}
 	curlArgs := []string{"--head", publicDNSQueryURI}
 	if proxyConfig.IsEnabled() {
-		if proxyConfig.HTTPProxy != "" {
-			curlArgs = append(curlArgs, "--proxy", proxyConfig.HTTPProxy)
-		}
+		proxyHost := proxyConfig.HTTPProxy
 		if proxyConfig.HTTPSProxy != "" {
+			proxyHost = proxyConfig.HTTPSProxy
+		}
+		if proxyHost != "" {
 			curlArgs = append(curlArgs, "--proxy", proxyConfig.HTTPSProxy)
```

Comment:
@cfergeau shouldn't it would be `curlArgs = append(curlArgs, "--proxy", proxyHost)`

Reply:
yes it should, thanks!